test/ui-testing/orgs.js

```diff
@@ -74,51 +61,59 @@ module.exports.test = (uiTestCtx) => {
       });
 
       orgs.forEach((org, row) => {
-        createOrg(org);
-
-        it('should add org line', done => {
+        it('should add org', done => {
           nightmare
-            .click('#add-license-org-btn')
+            .click('#add-org-btn')
             .evaluate((r) => {
-              if (!document.querySelector(`#org-name-${r}`)) {
-                throw Error('Expected name dropdown to exist.');
+              if (!document.querySelector(`#orgs-nameOrg-${r}-search-button`)) {
+                throw Error('Expected organization picker button to exist.');
               }
 
-              if (!document.querySelector(`#org-role-${r}`)) {
+              if (!document.querySelector(`#orgs-role-${r}`)) {
                 throw Error('Expected role dropdown to exist.');
               }
             }, row)
             .then(done)
             .catch(done);
         });
 
-        it(`should select org: ${org.name}`, done => {
+        it('should select org', done => {
           nightmare
-            .click(`#org-name-${row}`)
-            .insert(`#sl-container-org-name-${row} input`, org.name)
-            .waitUntilNetworkIdle(2000)
-            .click(`#sl-container-org-name-${row} ul li[aria-selected=false]`)
-            .evaluate((r, o) => {
-              const nameElement = document.querySelector(`#org-name-${r}`);
-              const name = nameElement.textContent;
-              if (name !== o.name) {
-                throw Error(`Expected name to be ${o.name} and is ${name}`);
+            .click(`#orgs-nameOrg-${row}-search-button`)
+            .wait('#clickable-filter-status-active')
+            .click('#clickable-filter-status-active')
+            .wait(`#list-plugin-find-organization [aria-rowindex="${row + 3}"] > a`)
+            .click(`#list-plugin-find-organization [aria-rowindex="${row + 3}"] > a`)
+            .evaluate((r, _orgs) => {
+              const orgElement = document.querySelector(`#orgs-nameOrg-${r}`);
+              const name = orgElement.value;
+              if (!name) {
+                throw Error('Org field has no value!');
               }
-            }, row, org)
+
+              return name;
+            }, row, orgs)
+            .then(name => {
+              orgs[row].name = name;
+            })
             .then(done)
             .catch(done);
         });
 
         it(`should assign role: ${org.role}`, done => {
           nightmare
-            .type(`#org-role-${row}`, org.role)
+            .waitUntilNetworkIdle(2000)
```

Comment:
What's the purpose of this `wUNI` and the one at the end? It doesn't seem like we're doing any network jobs in this case.